test_integration/README.md

```diff
@@ -0,0 +1,123 @@
+# Ably Realtime integration tests
+
+This is for tests that depend on the Ably plugin to be available and
+functional and for tests to be executed on Android and iOS devices or 
+emulators.
+
+## Execute the Flutter Driver Tests
+
+Run the bash script in the project root directory
```

Comment:
Again, if you remove the `cd` commands from the script then this can simplify.

Reply:
Done